src/main/java/com/treasuredata/client/TDClientConfig.java

```diff
@@ -95,6 +96,7 @@
     public final Optional<String> password;
     public final Optional<ProxyConfig> proxy;
     public final boolean useSSL;
+    public final String retryStrategy;
```

Comment:
If we use RetryStrategy here instead of String, we can validate the config in an early timing.